core/src/crds_gossip_pull.rs

```diff
@@ -371,10 +372,20 @@ impl CrdsGossipPull {
         &self,
         crds: &Crds,
         filters: &[(CrdsValue, CrdsFilter)],
+        now: u64,
     ) -> Vec<Vec<CrdsValue>> {
         let mut ret = vec![vec![]; filters.len()];
+        let msg_timeout = CRDS_GOSSIP_PULL_CRDS_TIMEOUT_MS;
         for v in crds.table.values() {
-            filters.iter().enumerate().for_each(|(i, (_, filter))| {
+            filters.iter().enumerate().for_each(|(i, (caller, filter))| {
```

Comment:
Can we add a metric to see how many these are filtering out? Unfortunately this existing counter: https://github.com/solana-labs/solana/blob/master/core/src/cluster_info.rs#L1831-L1834 wont catch these dropped values because it only takes the delta from all the pull responses generated *after* `generate_pull_responses()` returns: https://github.com/solana-labs/solana/blob/master/core/src/cluster_info.rs#L1774-L1794